### PR TITLE
Added Anagram Checker: #358

### PR DESCRIPTION
**Pull request by Anshika14528**
Closes: #358 

- **Title** : Anagram Checker
- **Name:** : Anshika
- **Idenitfy yourself:** : GSSOC

### Describe the add-ons or changes you've made 📃
Added Anagram Checker which tells us if two strings are anagram or not.

### How Has This Been Tested? ⚙️
I have testes it on my own.

### Screenshots 📷
Screenshot 2024-06-21 at 8 20 16 PM
screenshotss

### Type of change ☑️

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Anshika14528** [comment]
@Avdhesh-Varshney sir, please review it

File: Vanilla-JS-Projects/Intermediate/Anagram-Checker/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+<!-- add the steps how to run the project -->
+- Fork this project and run the `index.html` file directly.
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+![image]()
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Anagram-Checker/README.md:48]: update the screenshot properly

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Anshika14528 keep it inside `Basic` folder go according to label mentioned on issues and enlist your project in the readme file of `Vanilla-JS`

**Comment by Anshika14528** [comment]
@Avdhesh-Varshney  sir, i have made the required changes please review it now

**Review by Avdhesh-Varshney — APPROVED** [review]
@Anshika14528 PR Approved 🎉

**Pull request merged**